common/src/main/java/io/druid/audit/AuditEntry.java

```diff
@@ -91,6 +90,12 @@ public DateTime getAuditTime()
     return auditTime;
   }
 
+  @Override
+  public int compareTo(AuditEntry otherEntry)
```

Comment:
Since this seems to only be used to sort for returning results, can you imply have a comparator parameter to the sort instead of making the compare intrinsic to the class?